src/shared.coffee

```diff
@@ -128,9 +140,6 @@ class JobCollectionBase extends Mongo.Collection
         level = if fatal then 'danger' else 'warning'
         @_createLogEntry msg, runId, level).bind(@)
 
-    # Call super's constructor
-    super collectionName, options
```

Comment:
No access to `@` is allowed before calling `super` in the constructor. This is a JavaScript limitation.